libs/libc/pthread/pthread_attr_getstack.c

```diff
@@ -38,8 +35,9 @@
  * Description:
```

Comment:
Function description still missing. Maybe we could use the documentation from the OpenGroup specification:
```suggestion
 * Description:
 *   The pthread_attr_getstack() function shall get the thread creation stack
 *   attributes stackaddr and stacksize from the attr object.
```